sw/applications/example_power_gating_core/example_power_gating_core.c

```diff
@@ -68,47 +73,58 @@ int main(int argc, char *argv[])
     rv_timer_arm(&timer_0_1, 0, 0, 1024);
     rv_timer_counter_set_enabled(&timer_0_1, 0, kRvTimerEnabled);
 
-    if (power_gate_core(&power_manager, kTimer_0_e, &power_manager_cpu_counters) != kPowerManagerOk_e)
+    CSR_CLEAR_BITS(CSR_REG_MSTATUS, 0x8);
+    if (power_gate_core(&power_manager, kTimer_0_pm_e, &power_manager_cpu_counters) != kPowerManagerOk_e)
```

Comment:
you a separate function for the external power domain counters

Reply:
Not sure I got it